monero-outreach-docs/translations/es/storyboard/randomX_es.md

```diff
@@ -0,0 +1,35 @@
+
+RandomX tiene dos modalidades con diferentes requerimientos y rendimientos de memoria. Modo rápido requiere de 2 GB de memoria compartida pero tiene 4-6 veces más del rendimiento del Modo ligero que solo requiere de 256 MB de memoria RAM. El Modo rápido está destinado a mineros dedicados. El Modo ligero está diseñado para permitir a los nodos completos validar bloques sin necesitar las 2+GB de memoria RAM, para que los dispositivos pequeños (cómo los computadores ARM con board único, cómo por ejemplo, el Rock64) puedan seguir siendo usados como nodos independientes.
+
+| CPU | Sistema Operativo | Núcleos | RAM | CryptoNight-R (v8) | RandomX Modo Rápido | RandomX Modo Ligero |
+|--|--|--|--|--|--|--|
+| AMD Ryzen 7 1700 | Ubuntu 16.04 | 8 | 16 GB DDR4 | 650 H/s | 4100 H/s | 620 H/s |
+| Intel Core i7-8550U | Windows 10 | 4 | 16 GB DDR4 | 240 H/s | 1700 H/s | 350 H/s |
+| Intel Core i3-3220 | Ubuntu 16.04 | 4 | 4 GB DDR3 | 75 H/s | 510 H/s | 150 H/s |  
+
+| GPU | Frecuencia | CryptoNight-R (v8) | RandomX |
+|--|--|--|--|
+| GTX 1660 Ti | 2070/13760 MHz | 626 H/s (98 W) | 660 H/s (103 W) |
+| GTX 1080 Ti | 1930/10010 MHz | 787 H/s (145 W) | 1136 H/s (190 W) |  
+
+## _Cooperación para RandomX_ 
+
+RandomX fue desarrollado para Monero por tevador, hyc, vielmetti, antanst y SChernykh. Ya otras organizaciones están interesadas en adoptarlo. [Arweave](https://www.arweave.org/), un habilitador de almacenaje sin servidor, le donó la auditoría de Trail Of Bits a la comunidad de Monero y estará implementando RandomX para su aplicación antes que Monero. Arweave provee una innovadora propuesta basada en criptomonedas para descentralizar en el largo plazo el almacenamiento de la información. La minería de Arweave depende tanto de la prueba-de-trabajo y en un nuevo concepto único para Arweave, prueba-de-acceso. 
+
+[Wownero](http://wownero.org/) también estará lanzando RandomX en su próxima actualización v0.6 y le llamará RandomWOW. El código de RandomX será actualizado despues de que las auditorías estén completas, para entonces habrá algunas diferencia en el código para el momento del fork en octubre. Otra diferencia es que RandomWOW tendrá un almacenamiento local menor interno para hashing, 1 MB en vez de 2 MB, un menor número de iteraciones en las ejecuciones de la máquina virtual (VM - siglas en inglés) y un incremento encadenado en las ejecuciones de la VM por hash para incrementar el costo programático de compilación para los GPUs. 
```

Comment:
+ _El código de RandomX será actualizado **después** de que las auditorías estén completas,..._

Isn't there a translation yet for _fork_?

Reply:
Yes, there is. We had spoken about "bifurcación", but some majority didn't want to use "bifurcación" because "fork",  meaning literally "bifurcación", in Monero meant a "fork update", even though we move from one chain to another, leaving the last chain behind. What do you suggest?
I can work with bifurcación and giving it the correct meaning in Spanish.

Reply:
Added "bifurcación" to the document.